debezium-core/src/main/java/io/debezium/schema/SchemaChangeEvent.java

```diff
@@ -37,7 +37,8 @@ public SchemaChangeEvent(Map<String, ?> partition, Map<String, ?> offset, String
         this.offset = Objects.requireNonNull(offset, "offset must not be null");
         this.database = Objects.requireNonNull(database, "database must not be null");
         this.schema = Objects.requireNonNull(schema, "schema must not be null");
-        this.ddl = Objects.requireNonNull(ddl, "ddl must not be null");
+        // DDL is not mandatory
+        this.ddl = ddl;
```

Comment:
Ok, so you plan to solely rely on `TableChanges` for SQL Server. That's cool, I guess.